### ISSUE TITLE
AI Upscaling Might Be Broken

### ISSUE THREAD

**Issue by enzyme69** [carson-katri/dream-textures#432]
### Description

Unable to Upscale, error received right after the extra files downloaded.

```
Fetching 14 files: 100%|████████████████████████| 14/14 [04:58<00:00, 21.32s/it]
Exception: Traceback (most recent call last):
  File "/Users/jimmygunawan/Library/Application Support/Blender/3.5/scripts/addons/dream_textures/generator_process/actor.py", line 273, in _receive
    for res in iter(response):
  File "/Users/jimmygunawan/Library/Application Support/Blender/3.5/scripts/addons/dream_textures/generator_process/actions/upscale.py", line 52, in upscale
    low_res_image = Image.fromarray(image)
  File "/Users/jimmygunawan/Library/Application Support/Blender/3.5/scripts/addons/dream_textures/.python_dependencies/PIL/Image.py", line 2982, in fromarray
    arr = obj.__array_interface__
AttributeError: 'str' object has no attribute '__array_interface__'


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/jimmygunawan/Library/Application Support/Blender/3.5/scripts/addons/dream_textures/generator_process/actor.py", line 111, in run_callbacks
    done_callback(self)
  File "/Users/jimmygunawan/Library/Application Support/Blender/3.5/scripts/addons/dream_textures/operators/upscale.py", line 114, in image_done
    tile: ImageUpscaleResult = future.result()
  File "/Users/jimmygunawan/Library/Application Support/Blender/3.5/scripts/addons/dream_textures/generator_process/actor.py", line 58, in result
    raise self._exception
AttributeError: 'str' object has no attribute '__array_interface__'

```

### Steps to Reproduce

Image Editor. Load an image. Upscale. Failed.

### Expected Behavior

Maybe I was doing it wrong, but I tried upscaling 512x512 to 2048x2048. I don't know if my iMac 8GB can do this.

### Addon Version

macOS (Apple Silicon)

### GPU

Apple Silicon

**Comment by Atair1** [comment]
Same here, deleted the .cache folder and addon, reinstalled but got the same error.. windows / 3080 / cuda

**Comment by mj3017** [comment]
I have experienced a similar situation on two different PC (both on Windows 10).
It seems that the addon is unable to download a model, I guess.

**Comment by carson-katri** [comment]
Upscaling is not functioning in 0.0.9. I believe this will be fixed with #410

**Comment by votizlov** [comment]
Fixed this in #478

**Comment by enzyme69** [comment]
I think I just figure out that we need "Source Image" in order to upscale?
